### DISCUSSION TITLE
Is it possible to check on workers remotely?

### DISCUSSION THREAD

**Discussion by oivvio** [rq/rq#1751] [Q&A]
This is not an issue but  a question, but github really doesn't have a mechanism for posing questions.

I'd love to be able to monitor workers remotely via something like Pingdom, so that I can be notified if my workers aren't up and running.

**Comment by selwin** [comment]
At the moment, there's no way to do this. I'm open to suggestions though :)

**Comment by cgthayer** [comment]
We've "solved" this by having our workers look for something special in the message and respond to it. We have a ping cron job that sends the message and times the reply (or times out). The result is logged with sentry.io as success or failure (though we have "ping success" set to ignore). Sentry is connected to our slack dev channel so we get ping failures in real-time there (as well as email).

Lots of ways to implement this, though it might be helpful to have something built in.

**Comment by ccrvlh** [comment]
@selwin suggestion to move this as Q&A to the `Discussions` board.

**Comment by selwin** [comment]
@lowercase00 converted to a discussion :)

**Comment by ccrvlh** [comment]
I usually do this through the queue size, if it's growing too much is possible that the workers are down, and just checking the worker data on Redis.

**Comment by selwin** [comment]
I think we can make this easier by creating a cli command `rq worker info worker_id`. If someone wants to take a stab at this, I'm open to accepting a PR for this.
